crates/cxx-qt-lib/src/gui/qpainter.rs

```diff
@@ -239,6 +244,18 @@ mod ffi {
         #[rust_name = "draw_line"]
         fn drawLine(self: Pin<&mut QPainter>, line: &QLine);
 
+        /// Draws a line defined by line.
+        #[rust_name = "draw_linef"]
+        fn drawLine(self: Pin<&mut QPainter>, line: &QLineF);
+
+        /// Draws the set of lines defined by the list lines using the current pen and brush.
+        #[rust_name = "draw_lines"]
+        fn drawLines(self: Pin<&mut QPainter>, lines: &QList_QLine);
+
+        /// Draws the set of lines defined by the list lines using the current pen and brush.
+        #[rust_name = "draw_linefs"]
```

Comment:
I put `draw_linefs` as the rust name because I believe aliasing the names is not supported, right? In Qt they are both `drawLines()`.

Reply:
Yes, they need to be different names, so `draw_linefs` is good :+1: